docs/dasharo-tools-suite/documentation.md

```diff
@@ -584,6 +584,11 @@ firmware.
 
 ### EC update
 
+!!! note
+
+    This section only applies to NovaCustom releases earlier than v2.0.0 and is
+    obesoleted by the new automatic EC firmware sync feature.
```

Comment:
```suggestion
    obsolete by the new automatic EC firmware sync feature.
```

Reply:
Changed to `replaced`, @BeataZdunczyk is that ok?